packages/vs-input/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vivintsolar-oss/web-vs-input",
-  "version": "1.1.5",
+  "version": "1.1.61",
```

Comment:
6?

Reply:
Yeah I published twice. Once without using lerna so it didn't show up in the commit history.